packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayer.java

```diff
@@ -227,7 +227,11 @@ public void onPlaybackStateChanged(final int playbackState) {
           @Override
           public void onPlayerError(@NonNull final PlaybackException error) {
             setBuffering(false);
-            if (eventSink != null) {
+            if (error.errorCode == PlaybackException.ERROR_CODE_BEHIND_LIVE_WINDOW) {
+              // See https://exoplayer.dev/live-streaming.html#behindlivewindowexception-and-error_code_behind_live_window
+              exoPlayer.seekToDefaultPosition();
+              exoPlayer.prepare();
+            } else if (eventSink != null) {
```

Comment:
Would we not still want to forward the video error to Dart?

Reply:
I think not, since it's basically un-actionable from Dart. It could make sense if there was an option to disable this error handling logic from Dart, but it would require a change in platform interface - this seems to be an overkill for this fix.

Reply:
Longer term it would make more sense to have no handling here, and forward more structured errors to the clients so they could decide how to handle them, but we can start with this and see if there are requests for more control.